src/mixins/filing-template-mixin.ts

```diff
@@ -1506,7 +1518,12 @@ export default class FilingTemplateMixin extends DateMixin {
         if (!party.mailingAddress.streetAddressAdditional) delete party.mailingAddress.streetAddressAdditional
         if (!party.mailingAddress.deliveryInstructions) delete party.mailingAddress.deliveryInstructions
       }
-
+      // If the partyType is organization, remove firstName, middleName and lastName
+      if (party.officer.partyType === PartyTypes.ORGANIZATION) {
+        delete party.officer.firstName
+        delete party.officer.lastName
+        delete party.officer.middleName
+      }
```

Comment:
Why do we need this? I think the BE ignores these properties when the type is organization...?

If this is needed, say why in your comment (line 1521).

Reply:
Also, please test correction and change filings, since they, too, use this method. (And mention it in Test Notes in your ticket in case Riyaz wants to test it, too.)

Reply:
Yes agree, this is not needed. I will remove it

Reply:
Fixed. Resolving.